setup.py

```diff
@@ -29,7 +29,10 @@
         "flake8-breakpoint>=1.1.0,<2",  # detect breakpoints left in code
         "flake8-print>=4.0.0,<5",  # detect print statements left in code
         "isort>=5.10.1,<6",  # Import sorting linter
-        "pandas-stubs==1.2.0.62",  # NOTE: Needed due to mypy types
+        "mdformat>=0.7.16",  # Auto-formatter for markdown
+        "mdformat-gfm>=0.3.5",  # Needed for formatting markdown
+        "mdformat-frontmatter>=0.4.1",  # Needed for formatting markdown
+        "mdformat-footnote>=0.1.1",  # Needed for formatting markdown
```

Comment:
Add some better descriptions here

Reply:
Done. Realized we didn't need the footnote one at this time.